### ISSUE TITLE
Dialogs don't account for scaled text

### ISSUE THREAD

**Issue by Jayonas** [cmdwtf/KeePassDiceware#15]
The controls in the Settings dialog (and Salt dialog) look overly crowded on my machine compared to the example in your Readme.

<img width="248" alt="DefaultSettings" src="https://user-images.githubusercontent.com/37936740/203877076-c215ada2-bf34-4972-83c1-b337988e3d70.png">

In my experience the usual cause of that is when the app isn't correctly accounting for scaled up text on high resolution monitors (on Windows 10 this is in Settings under System => Display).

<img width="258" alt="Screenshot 2022-11-24 191800" src="https://user-images.githubusercontent.com/37936740/203877397-e51029da-20f8-46a4-b7c5-146113b10c6a.png">

Often the UI framework's layout system (e.g. WinForms, WPF) will handle this automatically but sometimes you need to set a flag or something.  Usually that's on the app level, though, and the rest of my KeePass interface looks fine, so I'm not sure what's different with your dialogs.  Maybe there's just something you need to turn on for them.

Alternatively, if you're doing your own math to layout the controls (and size the dialog) then you might need to fix your math to account for the scaling.  There should be some APIs in the UI framework that you can use for that, but it's been long enough that I've forgotten the details, sorry.

Thanks for the cool plugin, I was somehow unaware of the Diceware method before.

**Comment by nitz** [comment]
Hmm, interesting. I've only tested at 200% scale on a 4K screen and it seemed fine on that, so I'm wondering why it didn't scale properly for 150%. I've got an idea as to what might fix it, but I've got to drag my TV back to my PC in order to test it :)

Don't mind my rambling below which are mostly notes to myself as I think through this.

My experience with DPI awareness is pretty slim, but in every case I've dealt with it before, it's been a bit of an "all or nothing", with the application at the executable level requiring a marker in the manifest to claim DPI awareness. After that (w/WinForms) it just cascades through any form.

There is a DPI scaling mode that individual forms can tweak, but I feel like I've left it alone in the past and had luck with it in other applications. Perhaps the KeePass options form uses a different mode and all child forms inherit that by default, and I need to manually override it.
